.tool-versions

```diff
@@ -0,0 +1,2 @@
+nodejs 22
+java temurin-23
```

Comment:
I've opted for going for the temurin distribution by default since I've read a couple of times that it is quite well recommended. It's free to use (personal and production), open source and has quite a big community support.